### PR TITLE
Deprecate TimeOptions

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
No breaking changes in this PR that just removes superficial uses of `TimeOptions` and related code in the `itinerary-body` demo code.

### CODE REVIEW

File: packages/core-utils/src/deprecated.js [hunk 1]
```diff
@@ -316,3 +316,17 @@ export function summarizeQuery(query, locations = []) {
     : require("./itinerary").toSentenceCase(query.mode);
   return `${mode} from ${from} to ${to}`;
 }
+
+export function getTimeZoneOffset(itinerary) {
```
Comment (miles-grant-ibigroup) [inline packages/core-utils/src/deprecated.js:320]: We should probably add a deprecation warning

```suggestion
export function getTimeZoneOffset(itinerary) {
logDeprecationWarning("getTimeZoneOffset");
```

**Review by miles-grant-ibigroup — APPROVED** [review]
Works great thanks for cleaning that one up! Just one small addition to fully deprecate the method

File: packages/core-utils/src/deprecated.js (hunk 1 above)
Comment (binh-dam-ibigroup) [reply packages/core-utils/src/deprecated.js:320]: Yes, good one, thank you (https://github.com/opentripplanner/otp-ui/pull/398/commits/57b99e9ca4c62ccf622b5d38a14be4d6b6754fc6).

**Review by daniel-heppner-ibigroup — APPROVED** [review]
LGTM

**Pull request merged**